content/blog/2017/05/2017-05-17-pipeline-dev-tools.adoc

```diff
@@ -0,0 +1,85 @@
+---
+layout: post
+title: "Pipeline Development Tools"
+tags:
+- blueocean
+- pipeline
+- tutorial
+
+author: lnewman
+---
+
+NOTE: This is a guest post by link:https://github.com/bitwiseman[Liam Newman],
+Technical Evangelist at link:https://cloudbees.com[CloudBees].
+
+== Pipeline as code, made easier
+
+I've only been working with Pipeline for about a year.
+Pipeline in and of itself is a huge improvement over old style Jenkins projects.
+As a developer, it has been so great be able work with Jenkins projects
```

Comment:
The repetition of "project" here seems awkward, perhaps s/jenkins projects/Jenkins Pipelines/